### PR TITLE
Allow --enable-redis-igbinary for php70-redis

### PR DESCRIPTION
**Pull request by jenssegers**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Copied code from `php56-redis.rb` to enable `--enable-redis-igbinary`. Feedback is welcome!

### CODE REVIEW

**Comment by javian** [comment]
This is not going to work due to the igbinary dependency which does not have a released version for php7. I'd suggest to wait until that is available.

**Comment by javian** [comment]
They seem to be getting closer to the first php7 release so hopefully we can include this soon.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
Still waiting for a stable release from project.

**Comment by moderndeveloperllc** [comment]
@javian [igbinary 2.0 ](https://pecl.php.net/package-changelog.php?package=igbinary&release=2.0.0) was released a few weeks ago. It doesn't have official support for php7.1, but I'm guessing it works as the [Travis builds are passing](https://travis-ci.org/igbinary/igbinary). It actually looks like [2.0.1](https://github.com/igbinary/igbinary/releases) is coming today as it was tagged an hour ago.

**Comment by javian** [comment]
@moderndeveloperllc yes you seem to be able to download it from here https://github.com/igbinary/igbinary/releases - would you or @jenssegers be willing to submit a pr to update igbinary so that we can get this merged ?

**Comment by moderndeveloperllc** [comment]
It looks like it was tried before and closed out: #3747. I'm pretty new to brew, so I'm not sure how to fix the issue brought up there. @ilovezfs, do you have ideas on how to update igbinary properly?

**Comment by javian** [comment]
@moderndeveloperllc we haven't received a response yes in this issue from OP so feel free to submit a PR for php7.

**Comment by acim** [comment]
I am trying to get this finished and then dependency can be fulfilled.

https://github.com/Homebrew/homebrew-php/pull/3844

**Comment by acim** [comment]
I tried this formula and with this small fix it worked (php7 added in the path):

`cp "#{Formula["igbinary"].opt_include}/php7/igbinary.h", "ext/igbinary/igbinary.h"`

**Comment by javian** [comment]
I'd rather use the symlink method as suggested. OP has not responded for a while though.

**Comment by acim** [comment]
Just to mention that 3.1.0 is released and it would be nice to use that one instead. It would be also nice to have a 7.1 formula.

**Comment by javian** [comment]
@acim feel free to submit a PR to address those two if you want and then we can close this.

File: Formula/php70-redis.rb [hunk 1]
```diff
@@ -16,9 +16,15 @@ class Php70Redis < AbstractPhp70Extension
   end
 
   def install
+    args = []
+    args << "--enable-redis-igbinary"
```
Comment (javian) [inline Formula/php70-redis.rb]: Can you please make this optional and instead add an optional dependency to igbinary not to force it on to everyone ?

**Comment by javian** [comment]
Closing due to no response.

**Pull request closed without merging**